File: sample_programs/python/convert_temp.py
# temperature conversion using python
# Fareinheit to celcius and vice versa


option = int(input("1.C to F\n2.F to C\nOPTION: "))

if option == 1:

    celcius = float(input("Enter celcius: "))
    farenheit = (celcius - 32) * (5/9) # logic
    print("Conversion result(in Farenheit): ",farenheit)

elif option == 2:

    farenheit = float(input("Enter farenheit: "))
    celcius = (farenheit * 9/5) + 32
    print("Conversion result(in Celcius): ", celcius)


# Contributed by Senthilnathan

File: sample_programs/C++/convert_temp.cpp
// C++ Program to convert celcius to farenheit and vice versa

#include <iostream>

// using the namespace directives
using std::cin;
using std::cout;
using std::endl;

int main()
{
    //  Declaration of variables
    float celcius, farenheit;
    int option;

    cout << "Enter your option\n1.Farenhit to Celcius\n2.Celcius to Farenhiet\nOPTION: ";
    cin >> option;

    if (option == 1)
    {
        cout << "Enter Farenhit: ";
        cin >> farenheit;

        celcius = (farenheit - 32) * (5.0 / 9.0); // test code
        // celcius = (farenheit - 32) * (5 / 9);
        cout <<  "The conversion result is(in Celcius): " << celcius << endl;
    }
    else if (option == 2)
    {
        cout << "Enter Celcius: ";
        cin >> celcius;

        farenheit = (celcius * 9 / 5) + 32;
        cout <<  "The conversion result is(in Farenheit): " << (float)farenheit << endl;
    }
    else
    {
        // Print incase of options other than 1,2.
        cout << "Invalid input!" << endl;
    }

    return 0;
}


// Contributed by Senthilnathan
